docs/meshstack.gcp.index.md

```diff
@@ -102,56 +102,14 @@ In order to associate created projects with a Billing Account, the replicator ne
 
 ### meshfed-service User
```

Comment:
d: the title is misleading now, right? should be "Authorizing the Service Account" or something?

Reply:
right makes more sense.